extensions-contrib/kafka-emitter/src/main/proto/DruidSegmentEvent.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
+import "google/protobuf/timestamp.proto";
+
+option java_multiple_files = true;
+option java_package = "org.apache.druid.emitter.kafka.proto";
```

Comment:
you put this in the "kafka" package, but I don't see why this should be specific to Kafka. All emitters usually share the same formats, so even if we keep it in this package for now, we can make this format independent of a specific emitter.